subt/main.py

```diff
@@ -17,22 +17,8 @@
 from subt.local_planner import LocalPlanner
 
 
-RADIUS = 0.6  # 0.9  # 1.0
-
-# Return with artifact configuration
-#SEARCH_TIME_BEGIN = timedelta(minutes=1)
-#SEARCH_TIME_END = timedelta(minutes=5)
-
-# No artifacts configuration
-SEARCH_TIME_BEGIN = timedelta(minutes=4)
-SEARCH_TIME_END = timedelta(minutes=4)
-
-RETURN_TIMEOUT = SEARCH_TIME_END + timedelta(minutes=10)  # ??
```

Comment:
what happened with this constant?

Reply:
It has been replaced with command line argument as discussed earlier.

Reply:
see the `self.timeout/2` and `self.timeout` uses in `play_system_track` - let me know if you want some adjustments

Reply:
I ment `RETURN_TIMEOUT` which was much larger than original return time "just in case" the robot gets stuck in the tunnel so that the program does not terminate somewhere in the middle of the tunnel ... so maybe we should have `EXTRA_RETURN_TIMEOUT` set to these 10 minutes and add it to `self.timeout`?

Reply:
I'll rework this part.

Reply:
So now `self.timeout` is indeed timeout for exploration. The timeout to go back home is `2.5*self.timeout` now.

Reply:
OK, thanks